Ports/libfuse/package.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='libfuse'
+version='3.16.2'
+files=(
+    "https://github.com/libfuse/libfuse/releases/download/fuse-${version}/fuse-${version}.tar.gz#f797055d9296b275e981f5f62d4e32e089614fc253d1ef2985851025b8a0ce87"
+)
+useconfigure='true'
+configopts=(
+    '--cross-file'
+    "${SERENITY_BUILD_DIR}/meson-cross-file.txt"
```

Comment:
fwiw, in most cases we do `--cross-file=${SERENITY_BUILD_DIR}/meson-cross-file.txt` to avoid the entries being weirdly split like this.